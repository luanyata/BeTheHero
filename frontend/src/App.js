import React from "react";
import "./global.css";
import Login from "./pages/Login";

function App() {
  return <Login />;
}

export default App;
