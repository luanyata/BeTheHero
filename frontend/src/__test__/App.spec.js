import React from "react";
import { shallow } from "enzyme";
import App from "../App";

describe("<App />", () => {
  it("should to render the <App /> component", () => {
    const wrapper = shallow(<App />);

    expect(wrapper.debug()).toMatchSnapshot();
  });
});
