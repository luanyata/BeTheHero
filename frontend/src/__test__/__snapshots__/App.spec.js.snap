// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should to render the <App /> component 1`] = `"<Routes />"`;
